app/views/job_templates/_import_job_template_modal.html.erb

```diff
@@ -0,0 +1,18 @@
+<!-- modal window -->
+<div class="modal fade" id="importJobTemplateModal" role="dialog" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" onclick="close_import_job_template_modal(); return false;"><span aria-hidden="true">&times;</span><span class="sr-only"><%= _('Close') %></span></button>
+        <h4 class="modal-title">Import Job Template</h4>
+      </div>
+      <div class="modal-body" style="overflow-y: auto; max-height: 500px;">
```

Comment:
`style="overflow-y: auto; max-height: 500px;"` gets duplicated with app/views/job_invocations/_preview_hosts_modal.html.erb maybe we could extract it to external CSS file?